visualize.py

```diff
@@ -36,3 +36,6 @@
         elif word == ":PAD:":
             break
         html_file.write('<font style="background: rgba(255, 255, 0, %f)">%s</font>\n' % (alpha, word))
+
+print("")
```

Comment:
Add an ‘\n’ here, avoid messy notice .

Reply:
What do you mean by "messy notice"? Instead of this line you can just add "\n" to the next:
`print("\nOpen ...`